client/src/pages/SignUp/SignUpForm/SignUpForm.tsx

```diff
@@ -51,9 +53,10 @@ const SignUpForm = ({ handleSubmit }: Props): JSX.Element => {
     >
       {({ handleSubmit, handleChange, values, touched, errors, isSubmitting }) => (
         <form onSubmit={handleSubmit} className={classes.form} noValidate>
+          <Typography className={classes.label}>Email address</Typography>
           <TextField
-            id="username"
-            label={<Typography className={classes.label}>Username</Typography>}
+            id="email"
+            placeholder="email"
```

Comment:
This is somewhat a preference issue, but I would suggest using placeholders such as `Your Email...` or `Enter a password...`.